02-peer-protocol.md

```diff
@@ -1344,7 +1396,8 @@ A node:
   sent and received:
     - MUST retransmit `funding_locked`.
   - otherwise:
-    - MUST NOT retransmit `funding_locked`.
+    - MUST NOT retransmit `funding_locked`, but MAY send `funding_locked` with
```

Comment:
I think this may be a backwards incompatible change unless we require the feature bit here - at least we used to check the `funding_locked` `next_per_commitment_point` by checking it against the channel's current commitment id, which will fail if a `funding_locked` is sent after the channel is in normal operation.